main.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'app'
+
+def print_menu
```

Comment:
- [ ] In this file, kindly create a method called ```main``` which should contain only the options the user has to input and the expected responses. This ```main``` method should be invoked at the end of the file as the requirement below states

![Screenshot 2023-02-17 at 09 15 22 (2)](https://user-images.githubusercontent.com/64774936/219591317-da458d1a-b7d9-44f0-928b-6829ba242008.png)